src/features/DisplayControls/NursingTasks/tests/AddEmergencyTasks.spec.jsx

```diff
@@ -170,11 +170,11 @@ describe("AddEmergencyTasks", () => {
     // Set Administration Date
     const datePickerInput = container.querySelector(".bx--date-picker__input");
     fireEvent.change(datePickerInput, {
-      target: { value: moment().format("DD MMM YYYY") },
+      target: { value: moment().format("DD/MM/YYYY") },
```

Comment:
you need to revert this change.... run yarn install and as we have updated the version of bahmni-carbon-ui, after that this test would pass with DD MMM YYYY format